Nos/Views/Components/Media/GalleryView.swift

```diff
@@ -0,0 +1,106 @@
+import SwiftUI
+
+/// Displays an array of URLs in a tab view with custom paging indicators.
+/// If only one URL is provided, displays an `ImageButton` or `LinkPreview` depending on the URL.
+struct GalleryView: View {
+    /// The URLs of the content to display.
+    var urls: [URL]
```

Comment:
What's your take on `var` vs `let` in View structs? Even though I always slightly preferred `var` because most SwiftUI examples at https://developer.apple.com/xcode/swiftui/ use `var`, my PRs have been corrected whenever I used them and asked me to use `let` instead.
I'm fine with either way given that even though you declare them as `var`, you cannot change them (SwiftUI doesn't let you). But, anyway, given that you used let in other places (like in `ImageButton` below), so we should come up with an arbitrary decision and use the same thing everywhere.

Reply:
I completely agree that we should establish a guideline and follow it. I don't know why I made this `var` -- my rule is that if it doesn't need to change, it should be `let`. Always.